### PR TITLE
negar/spinner_style

### PR DESCRIPTION
**Pull request by negar-binary**
change to new spinner style from binary style

### CODE REVIEW

File: src/templates/bet/mb_trading.html.tt [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <div class="center-text">
-    <img class="spinner" src="[% request.url_for('images/pages/binary-symbol-logo.svg') %]" alt="" />
+    <div class="barspinner dark">
+        <div class="rect1"></div>
+        <div class="rect2"></div>
+        <div class="rect3"></div>
+        <div class="rect4"></div>
+        <div class="rect5"></div>
+    </div>
```
Comment (binary-hamedanchi) [inline src/templates/bet/mb_trading.html.tt]: Is it possible to put this part in a separate template file, and just include it everywhere needed?

Comment (negar-binary) [inline src/templates/bet/mb_trading.html.tt]: @binary-hamedanchi I have made the change, thank you.

**Comment by jy-binary** [comment]
pls fix conflicts

**Comment by negar-binary** [comment]
@jy-binary thank you, conflicts are resolved

**Pull request merged**